app/code/Magento/Theme/Block/Html/Header/Logo.php

```diff
@@ -101,6 +102,19 @@ public function getLogoWidth()
         return (int)$this->_data['logo_width'];
     }
 
+    /**
+     * Retrieve logo width
+     * @return int
+     */
+    public function getLogoImageWidth()
+    {
+        $logoWidth = $this->_scopeConfig->getValue(
+            'design/header/logo_width',
+            \Magento\Store\Model\ScopeInterface::SCOPE_STORE
+        );
+        return $logoWidth ? (int) $logoWidth : $this->_data['logo_width'];
```

Comment:
Isn't that better to cast type to int for whole line just in case there's string in _data?